airbyte-local.sh

```diff
@@ -61,6 +60,15 @@ function parseFlags() {
             --check-connection)
                 check_src_connection=true
                 shift 1 ;;
+            --skip-src-pull)
+                skip_src_pull=true
+                shift 1 ;;
+            --skip-dst-pull)
+                skip_dst_pull=true
+                shift 1 ;;
+            --origin)
```

Comment:
it's better be called stream_prefix (to follow the airbyte terminology), no?!

Reply:
But it's not actually the stream prefix, it's just one component of it. I guess a more accurate name would be `connection_name`.